### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by ntzwilly**
# Project 4: Desktop version

   Use media queries to make sure that the website displays correctly on desktop screens.

## Project requirements: 

- Use media queries in the CSS file to make the desktop version look like the template you selected.

- Use Grid and Flexbox in order to layout the elements on the page.

- No need to implement any functionality that requires JavaScript.

- Replicate the design as much as possible (e.g., font, colors, images, tests, margins) using the info from the Figma templates.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,256 +76,302 @@
...
+        <input
+          name="text"
+          class="text-input"
+          type="text"
+          maxlength="30"
+          placeholder="Full name"
+          required
+        /><br />
+        <input
+          name="email"
```
Comment (Temmarie) [inline index.html:359]: - [ ] The background image here has some placement issues. Kindly fix it.

_Refer to my comments on the design above._

_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @ntzwilly and @Munsa1    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️
- [x] Kindly refer to the design in Figma and try to make the project as similar as possible to the template you have chosen.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html [hunk 2]
```diff
@@ -32,6 +32,13 @@
...
           <nav class="hamburger-container"><i class="fas fa-bars"></i></nav>
+          <nav class="header-nav-links">
+            <ul>
+              <li><a href="#">Portoflio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </nav>
         </div>
       </header>
```
[the remark this answers is not in the document]
Comment (ntzwilly) [reply index.html:43]: Thank you for the review, this issue has been solved

File: index.html [hunk 3]
```diff
@@ -23,7 +23,7 @@
     <title>Portfolio</title>
   </head>
   <body>
-    <main>
+    <main class="main-background">
       <header>
         <div class="header-container">
           <div class="logo-container">
```
[the remark this answers is not in the document]
Comment (ntzwilly) [reply index.html:29]: I added the remaining sections to the main and used the images from the Figma template. Also used typography from Figma

File: index.html [hunk 4]
```diff
@@ -69,256 +76,302 @@
...
+        >
+        <a class="link" href="#"><h4 class="header-2">LET’S CONNECT</h4></a>
+        <ul class="nav-link">
+          <li class="circle">
+            <a href="#"><img src="images/git.svg" alt="git-logo" /></a>
+          </li>
+          <li class="circle">
+            <a href="#"><i class="fab fa-linkedin-in"></i></a>
+          </li>
+          <li class="circle">
```
[the remark this answers is not in the document]
Comment (ntzwilly) [reply index.html:282]: This issue is now solved

File: index.html (hunk 1 above)
Comment (ntzwilly) [reply index.html:359]: The background is now well placed

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Dipeshtwis) [reply index.html]: This issue was successfully addressed. :+1:

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (Dipeshtwis) [reply index.html:291]: This issue was successfully addressed. :+1:

File: index.html (hunk 1 above)
Comment (Dipeshtwis) [reply index.html:368]: This issue was successfully addressed. :+1:

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @ntzwilly, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dipeshtwis) [reply index.html:29]: This issue was successfully addressed. :+1:

**Pull request merged**